editor/editor_properties.cpp

```diff
@@ -3701,6 +3673,35 @@ void EditorPropertyNodePath::_notification(int p_what) {
 
 void EditorPropertyNodePath::_bind_methods() {
 }
+Node *EditorPropertyNodePath::get_base_node() {
```

Comment:
Missing a new line, but also this method should be somewhere above, among other similar methods.

Reply:
Did the change so we can merge this now.

Reply:
I don't see any changes 👀

Reply:
Welp, seems like I forgot to force push?